packages/frontend/src/Build.svelte

```diff
@@ -0,0 +1,370 @@
+                  href="https://github.com/containers/podman-desktop-extension-bootc">README</a
+                >.
+              </p>
+            {/if}
+            <p class="text-gray-300 text-xs pt-1">
+              Note: All images must be pushed to a publically accessible registry in order for our build system to
+              "pull" the images. This is TEMPORARY and local image building will be supported in the future.
+            </p>
+          </div>
+          <div>
+            <label for="path" class="block mb-2 text-sm font-bold text-gray-400">Build output folder</label>
+            <div class="flex flex-row space-x-3">
+              <Input
+                name="path"
+                id="path"
+                bind:value="{buildFolder}"
+                placeholder="Output folder"
+                class="w-full"
+                aria-label="folder-select" />
+              <Button on:click="{() => getPath()}">Browse...</Button>
+            </div>
+          </div>
+          <div class="pt-3 space-y-7 h-fit">
+            <div class="mb-2">
+              <span class="text-md font-semibold mb-2 block">Image type</span>
+              <div class="flex items-center mb-3">
+                <label for="raw" class="ml-1 flex items-center cursor-pointer">
+                  <input
+                    bind:group="{buildType}"
+                    type="radio"
+                    id="raw"
+                    name="format"
+                    value="raw"
+                    class="sr-only peer"
+                    aria-label="raw-select" />
+                  <div
+                    class="w-4 h-4 rounded-full border-2 border-gray-400 mr-2 peer-checked:border-purple-500 peer-checked:bg-purple-500">
+                  </div>
+                  <span class="text-sm text-white">RAW image with partition table (*.raw)</span>
+                </label>
+              </div>
+              <div class="flex items-center mb-3">
+                <label for="qcow2" class="ml-1 flex items-center cursor-pointer">
+                  <input
+                    bind:group="{buildType}"
+                    type="radio"
+                    id="qcow2"
+                    name="format"
+                    value="qcow2"
+                    class="sr-only peer"
+                    aria-label="qcow2-select" />
+                  <div
+                    class="w-4 h-4 rounded-full border-2 border-gray-400 mr-2 peer-checked:border-purple-500 peer-checked:bg-purple-500">
+                  </div>
+                  <span class="text-sm text-white">Virtualization Guest Image (*.qcow2)</span>
+                </label>
+              </div>
+              <div class="flex items-center mb-3">
+                <label for="iso" class="ml-1 flex items-center cursor-pointer">
+                  <input
+                    bind:group="{buildType}"
+                    type="radio"
```

Comment:
You can switch to the Input component provided by the library https://www.npmjs.com/package/@podman-desktop/ui-svelte